src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -300,6 +293,7 @@ export function getStatusContents(statusType, details = {}, name = {}) {
       break;
     case STATUS_TYPES.bvaDevelopment:
       contents.title = 'The judge is seeking additional information before making a decision';
+      // TODO: Should this be "to make A decision"?
```

Comment:
@gnakm should the description here read "make a decision" rather than "make decision"?

Reply:
I'm slightly confused. Are you talking about line 295 because it says The judge is seeking additional information before `making a decision`

Reply:
Thanks for following up @gnakm, the question relates to the following line: 
```
       contents.description = (
         <p>The Board of Veterans’ Appeals is seeking evidence or an outside opinion from a legal,
         medical, or other professional necessary to make decision about your appeal.</p>
```

Reply:
`make a decision` please !

Reply:
Thanks @gnakm, updated.